apps/src/weblab/WebLab.js

```diff
@@ -444,8 +448,10 @@ WebLab.prototype.loadFileEntries = function () {
       // current version (until the browser page reloads)
       project.filesVersionId = result.filesVersionId;
     }
-    if (this.brambleHost) {
+    if (this.brambleHost && this.brambleReady) {
       this.brambleHost.syncFiles(() => {});
+    } else {
+      this.syncBrambleWhenReady = true;
```

Comment:
Hmm... Having parallel asynchronous dependencies like this makes me think this is a `Promise.all` situation, though I suppose we have some hoops to jump through for that.  Could we do something like this?

```js
// @ WebLab.prototype.init where we currently call loadFileEntries
const fileEntriesLoaded = this.loadFileEntries();
const brambleReady = new Promsie((resolve) => this.reportBrambleReady = resolve);
Promise.all([fileEntriesLoaded, brambleReady]).then(() => {
  this.brambleHost.syncFiles(() => {});
});

// @ WebLab.prototype.setBrambleHost calls reportBrambleReady when Bramble is ready

// @ WebLab.prototype.loadFileEntries updated to return a Promise
```